README.md

```diff
@@ -1,16 +1,11 @@
 # The JFK Files
-Explore the JFK Assassination files using the Cognitive Search features of Azure Search.
-You can watch the demo in action in a short [online video](https://channel9.msdn.com/Shows/AI-Show/Using-Cognitive-Search-to-Understand-the-JFK-Documents)
-or explore the JFK files yourself with our [online demo](https://aka.ms/jfkfiles-demo).
+Explore the JFK Assassination files using the Cognitive Search features of Azure Search. You can watch the demo in action in a short [online video](https://channel9.msdn.com/Shows/AI-Show/Using-Cognitive-Search-to-Understand-the-JFK-Documents) or explore the JFK files yourself with our [online demo](https://aka.ms/jfkfiles-demo).
```

Comment:
Looks like we never properly rebranded this repo for Azure Cognitive Search and this is now making me notice that.  @LuisCabrer do you know if someone who was involved in the rebrand could look over this README and make sure the right names are used throughout?

Reply:
I don't believe anyone did Carey.
The text above should read:  "Explore the JFK Assassination files using the AI Enrichment features of Azure Cognitive Search. You can watch the demo in action in a short..."

Reply:
I know it wasn't part of this PR, but I updated that text to match this suggestion as well.